src/utils/FetchConfig.ts

```diff
@@ -83,6 +83,13 @@ export async function FetchConfig (): Promise<any> {
   const keycloakClientId: string = import.meta.env.VUE_APP_KEYCLOAK_CLIENTID;
   (<any>window).keycloakClientId = keycloakClientId
 
+  // Document Record Service Configration
+
+  const docApiKey: string = import.meta.env.VUE_APP_DOC_API_KEY
+  sessionStorage.setItem('DOC_API_KEY', docApiKey)
```

Comment:
I recommend not storing the API key in session storage. Instead, you can get this from import.meta.xxx right where you actually use it.

Reply:
I think you've made the change to the bcrs-shared component that needs this API key, right? Can these 2 lines be removed now?